### DISCUSSION TITLE
RasterizingTileLayer click feature event does not work

### DISCUSSION THREAD

**Discussion by PopescuGeorge94** [Mapsui/Mapsui#2177] [General]
Hi, I have the code to render a geojson.

            var geojsonProvider = new GeoJsonProvider(geoJson)
            {
                CRS = "EPSG:3857" 
            };            

            var layer = new RasterizingTileLayer(new Layer
            {
                Name = "Polygons",
                DataSource = new GeometrySimplifyProvider(geojsonProvider),
                Style = CreateStyle(),
                MaxVisible = 21D,
                FetchingPostponedInMilliseconds = 500,
                IsMapInfoLayer = true
            });

When I use RasterizingLayer I can catch the click event, but when I use RasterizingTileLayer I can't do that anymore.

**Comment by inforithmics** [comment]
How do you catch the click event?

**Reply by PopescuGeorge94** [reply]
mapControl.Map.Info += (s, a) =>
  {
     
  };

**Reply by inforithmics** [reply]
I had a look at it RasterizingTileLayer and RasterizingLayer need to implement IFeatureInfo to achieve this functionality
